src/frontend/components/TeacherLiveRecordingActions/StopRecording/index.tsx

```diff
@@ -0,0 +1,92 @@
+import { defineMessage } from '@formatjs/intl';
+import { Box, Button, Clock, Spinner } from 'grommet';
+import React, { useRef } from 'react';
+import toast from 'react-hot-toast';
+import { useIntl } from 'react-intl';
+
+import { PauseSVG } from 'components/SVGIcons/PauseSVG';
+import { useStopLiveRecording } from 'data/queries';
+import { Video } from 'types/tracks';
+import { Maybe } from 'utils/types';
+
+const formatDigits = (value: number, locale: string) => {
+  return value.toLocaleString(locale, {
+    minimumIntegerDigits: 2,
+    useGrouping: false,
+  });
+};
+
+const formatSecToTimeStamp = (duration: Maybe<number>, locale: string) => {
+  let recordedTime = 'T00:00:00';
+  if (duration) {
+    const sec = formatDigits(duration % 60, locale);
+    const min = formatDigits(Math.floor(duration / 60) % 60, locale);
+    const hour = formatDigits(Math.floor(duration / 3600), locale);
+
+    recordedTime = `T${sec}:${min}:${hour}`;
```

Comment:
```suggestion
    recordedTime = `T${hour}:${min}:${sec}`;
```

You should provide a test verifying this function.

Reply:
[7076eb6](https://github.com/openfun/marsha/pull/1301/commits/7076eb631b4364f704a1f911db4e030e95c023f1)